CatAnalyzer/interface/LepJetsFitter.h

```diff
@@ -34,7 +34,7 @@ namespace ttbb{
   TLorentzVector tmplep, tmpnu, tmpbl, tmpbj, tmpj1, tmpj2;
   float blres, bjres, j1res, j2res, metres;
 
-  float CSVWP = 0.800;
+  float CSVWP = 0.9535;
```

Comment:
You can take the WP values from the header file.
https://github.com/vallot/CATTools/blob/cat80x/DataFormats/interface/Jet.h#L34